functions/gcleanx.fish

```diff
@@ -0,0 +1,3 @@
+function gclean!! -w "git clean -dfx" -d "Pristine git working directory: reset and force clean"
```

Comment:
I'm worried the filename not matching the function name will cause this to not be loaded: https://fishshell.com/docs/current/#autoloading-functions

Given this function doesn't do anything other than run two commands, we should be able to convert it to an abbreviation.

Example: `__git_abbr gfm        "git fetch origin master --prune; and git merge FETCH_HEAD"`

Reply:
Agree, I have done some refactor. Please have a look if you get some time. Thanks.